src/@types/parseable/api/stream.ts

```diff
@@ -7,6 +7,11 @@ export type LogStreamSchemaData = {
 	metadata: Record<string, string>;
 };
 
+export type LogStreamQueryWithFields = {
+	fields: string[];
+	records: string[];
```

Comment:
Dont think this is an array of strings. 
These are log records. This type needs to be fixed.